examples/multiple-domains-same-records/main.tf

```
@@ -5,23 +5,12 @@
 # ---------------------------------------------------------------------------------------------------------------------
 
 # ---------------------------------------------------------------------------------------------------------------------
-# Set Terraform and Provider Requirements for running this example
-# ---------------------------------------------------------------------------------------------------------------------
-
-terraform {
-  required_version = ">= 0.12.20"
-
-  required_providers {
-    aws = ">= 2.45"
-  }
-}
-
-# ---------------------------------------------------------------------------------------------------------------------
-# CONFIGURE OUR AWS CONNECTION
+# Configure the AWS Provider
 # ---------------------------------------------------------------------------------------------------------------------
 
 provider "aws" {
-  region = "us-east-1"
+  region  = "us-east-1"
+  version = ">= 2.45"
```

```suggestion
  version = "~> 2.45"
```

please change in all places... else 3.0 release can break all tests..

---

resolved in 9605794